visualization/eolearn/visualization/eopatch_base.py

```diff
@@ -78,7 +78,7 @@ def __init__(
 
         if rgb and len(rgb) != 3:
             raise ValueError(f"Parameter rgb should be a list of 3 indices but got {rgb}")
```

Comment:
Should this be 

```suggestion
            raise ValueError(f"Parameter rgb should be a list of 3 indices but got {len(rgb)}")
```

?